test/metabase/task/sync_databases_test.clj

```diff
@@ -108,18 +108,18 @@
 (deftest schedule-changes-only-expected-test
   (is (= [sync-job
```

Comment:
I guess it doesn't really make a bug difference here but I think moving the `=` forms in as deep as possible is preferable. If we wanted to tweak that `with-temp-scheduler` macro to generate a `testing` form we'd be able to take advantage of that extra context. A lot of our other test macros like `dataset` or `with-temporary-setting-values` do that. It makes adding new assertions next to the original ones easier too

Reply:
i kinda agree with you but i suspect i know what rob is doing. putting the `is =` forms above all of the context setup lets you easily eval the forms more easily. you can run the tests in cider with `C-c C-t t` and get the nice diff but sometimes its nice to get the value in the repl to print, save, drill around etc. And to do this otherwise requires raising the `(is (= expected (form-i-actually-want)))` out of the other things

Reply:
That's a good point 👍